### PR TITLE
doc: prepare new release_3.3.1

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
Related to #3952 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] If there are modifications in the Tracim configuration files (eg. `development.ini`), make sure it is documented in `backend/doc/setting.md` and notify relevant people of this change

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- No manual, quality tests necessary

### CODE REVIEW

**Comment by PhilippeAccorsi** [comment]
Waiting modification for 3.3.0 (add new feature).

File: changelog.md [hunk 1]
```diff
@@ -1,10 +1,21 @@
 
 
+## 3.3.1 / 2020-12-17
+
+### Fixed Issues
+
+- Note: [#3722](https://github.com/tracim/tracim/issues/3722)
+- Mention: [#3927](https://github.com/tracim/tracim/issues/3927)
+- Frontend: [#3542](https://github.com/tracim/tracim/issues/3542),
+[#3809](https://github.com/tracim/tracim/issues/)
```
Comment (inkhey) [inline changelog.md:10]: wrong url.

**Review by inkhey — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 1 above)
Comment (PhilippeAccorsi) [reply changelog.md]: Done.

File: changelog.md [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
+
+
 ## 3.3.0 / 2020-12-02
 
 ### New Features
 
 - Personal and per-space activity feeds help you keep up with what is going on in Tracim: you will be able to notice new contents, new members and other updates in real time at a glance.
+- Access to applications through tabs (dashboard, contents, activity feed, etc)
```
Comment (raphj) [inline changelog.md:18]:
```suggestion
- Tabs to access the dashboard contents and activity feed
```

(There is nothing else for now, I think it's clearer like this)

**Review by raphj — APPROVED** [review]

File: changelog.md [hunk 3]
```diff
@@ -1,10 +1,21 @@
 
 
+## 3.3.1 / 2020-12-17
+
+### Fixed Issues
+
+- Note: [#3722](https://github.com/tracim/tracim/issues/3722)
+- Mention: [#3927](https://github.com/tracim/tracim/issues/3927)
```
Comment (gbonaspetti) [inline changelog.md:8]:
```suggestion
- Mention: 
    - [#3722](https://github.com/tracim/tracim/issues/3722)
    - [#3927](https://github.com/tracim/tracim/issues/3927)
```

File: changelog.md [hunk 4]
```diff
@@ -1,10 +1,21 @@
...
+- Frontend: [#3542](https://github.com/tracim/tracim/issues/3542),
+[#3809](https://github.com/tracim/tracim/issues/3809)
```
Comment (gbonaspetti) [inline changelog.md:10]:
```suggestion
- Frontend: 
    - [#3542](https://github.com/tracim/tracim/issues/3542),
    - [#3809](https://github.com/tracim/tracim/issues/3809)
```

File: changelog.md (hunk 3 above)
Comment (PhilippeAccorsi) [reply changelog.md:8]: At present, we use this syntax to show issue like this

Screenshot_2020-12-17 tracim tracim

In futur, we can show issue with your idea to add issue description (not available for the moment). This change need to be done in all changelog.

Comment (PhilippeAccorsi) [reply changelog.md:8]: I just seen you add 3722 in mention and not in Note, why ? Is it more related to mention ?

File: changelog.md (hunk 4 above)
Comment (PhilippeAccorsi) [reply changelog.md:10]: same as explain before (this is not the syntax for the chnagelog, at present)/

File: changelog.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply changelog.md:18]: @raphj I'm not sure about this. Tabs not just shown dashboard content and activity feed, so I think its better to find wording to explain the futur tabs can be added in this view.

File: changelog.md (hunk 3 above)
Comment (gbonaspetti) [reply changelog.md:8]: This way it's good too, my suggestion was more to put the issue as mention and not note. I think is more related.

File: changelog.md (hunk 4 above)
Comment (gbonaspetti) [reply changelog.md:10]: good for me, I'll resolve the conversation.

File: changelog.md (hunk 3 above)
Comment (PhilippeAccorsi) [reply changelog.md:8]: Ok I put issue in Mention. Thanks.

**Pull request merged**